models/dataset.py

```diff
@@ -0,0 +1,193 @@
+"""Open datasets and processes them to be used by a neural network"""
+
+
+import os
+import numpy as np
+import h5py
+import torch
+from torch.utils.data import DataLoader, random_split
+import functools
+from PIL import Image
+
+CUDA = torch.cuda.is_available()
+
+if CUDA:
+    CRYO_TRAIN_VAL_DIR = os.getcwd() + "/Cryo/VAE_Cryo_V3/Data/"
+else:
+    CRYO_TRAIN_VAL_DIR = os.getcwd() + "\\Data\\"
```

Comment:
Is this absolutely needed?

Reply:
this is the path to data

Reply:
OK we can fix that later. 

I have two more specific comments:
- It does not appear very clearly why the path is dependent on the availability of torch.cuda
- I doubt the data will be accessible to any user which means most users won't be able to use this as it is